src/shared/route-urls.ts

```diff
@@ -31,7 +31,7 @@ export enum RouteUrls {
   IncreaseStxFee = '/increase-fee/stx',
   IncreaseBtcFee = '/increase-fee/btc',
   IncreaseFeeSent = '/increase-fee/sent',
-  Send = '/send-transaction',
+  // Send = '/send-transaction', // cannot find usage
```

Comment:
Pretty sure this is an app-initiated tx flow when sending stx?

Reply:
As in, external apps need to use this? I will re-add it as a precaution. 

I just couldn't find us using it anywhere in the extension code.